### ISSUE TITLE
Overhaul the help pages, to make them shorter, prettier and more accessible.

### ISSUE THREAD

**Issue by FOIMonkey** [mysociety/whatdotheyknow-theme#1453]
### What's the issue?

- ❗The current help pages are very long - in total they include 20,270 words.
- ❗There is no proper index/toc
- ❗Although we can sometimes link to individual headings, there is no easy way to jump between these.
- ❗A lot of the language we use is quite formal. 
- ❗With the exception of the Pro page, there are no images.

### Why is this a problem?

- ❌ Walls of text are intimidating for lower literacy users & second language speakers.
- ❌ If the words that we use are hard to understand, this could increase misuse
- ❌ The longer each page is, the less likely it is that anyone will read it.
- ❌ It's hard to find what you are looking for without using the browser search
- ❌ A lack of images or other elements to break up the text makes it harder to read, and visually unappealing.

### Possible solutions:

- ✅ Completely overhaul the help section and start again from the beginning. Instead of being restricted by the current structure, we could think about who the audience is that want to target with each part, or deliver the help pages by theme.
- ✅ Split the current help pages into subpages, with proper navigation links to move between them.
- ✅ Collapse the text that's not the headline/question on the longer help pages to make them seem shorter.
- ✅ Add images, or other styling to break up the wall of text.

### A more ordered International example:
🏴‍☠️ https://fragdenstaat.de/hilfe/
🏴‍☠️ https://fragdenstaat.de/en/about-us/netiquette/

Ideally I'd love to see us do this before adding yet more stuff.

**Comment by FOIMonkey** [comment]
This could unlock/enable/close #1443, #1422, #1442, #1353, #1217, #1157, #1140, #1065, #1054, #1050, #1027,,#1026,#1021,
#996, #994, #991, #987, #984, #976, #952, #914, #906, #905, #878, #877, #841, #790, #787, #786, #728, #621 and many more!

**Comment by WilliamWDTK** [comment]
A lot of sites have a sort of knowledgebase structure, with each page only being one or two paragraphs long, with links between them, and pages being in a category etc, and having a dedicated search facility.

**Comment by RichardTaylor** [comment]
Addition to:

** Why is this a problem? **

* ❌ Our help pages are the primary place we record our policies and practices for running the service. We want it to be easy for new administrators to find, understand and apply our policies.

**Comment by mdeuk** [comment]
> > Emoji
> 
> We should consider accessibility.
> 
> I'm more of a fan of the icons used by FragDenStaat over Emoji, I am in favour of [more illustration](https://github.com/mysociety/alaveteli/issues/6586).

As am I - it’s the “what’s doable with our limited resources” that might cause some issue here (albeit, we might be able to use Bootstrap icons](https://icons.getbootstrap.com/) for some).

If we were using Emoji, I would do it roughly in the same way that @FOIMonkey’s original message had - albeit, you could use them in less places. It’s one to ponder as a starter for ten.

**Comment by mdeuk** [comment]
Thinking of how icons can be used to represent content in a more digestable format, I spotted this sign at a Network Rail managed station.

In essence, it's a very simplified TL;DR of their Privacy Notice… with a link back to the weightier version.

<details>
<summary>Photograph of Privacy Notice sign</summary>
<img src="https://user-images.githubusercontent.com/249418/242834512-2af19af4-641f-4372-8be9-7b8bbbc81fb6.jpg">
</details>

Being able to have a "what you _need_ to know" summary at the top of our policy docs would help make them easier to understand. After all, do we really think that many folks would sit and read the whole thing? Given [ToS;DR exist](https://tosdr.org/en/about)s, I suspect it is a real issue.

See also, this [excellent version from Twitter](https://web.archive.org/web/20230602084902/https://twitter.com/en/privacy). The policy itself is probably less excellent, but the TL;DR at the top gives you the _essential information_, so you can hopefully get the salient points quickly.

**Comment by HelenWDTK** [comment]
This issue is being closed due to a lack of discussion or resolution for over 12 months. Should we decide to revisit this issue in the future, it can be reopened.
